frontend/src/container/NewDashboard/DashboardVariablesSelection/VariableItem.tsx

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable sonarjs/cognitive-complexity */
```

Comment:
can we remove the global disable over a file 

also can we try to remove the cognitive-complexity

Reply:
This limit was pretty small and didn't always make sense because purely based on the number of lines. I will move the specific function.